assemblies/setting-up-trustyai-for-your-project.adoc

```diff
@@ -0,0 +1,31 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+
+[id="setting-up-trustyai-for-your-project_{context}"]
+= Setting up TrustyAI for your project
+
+[role='_abstract']
+
+To set up model monitoring with TrustyAI for a data science project, a data scientist does the following tasks:
```

Comment:
Same kind of comment as before, if this is aimed at data scientists, should we not use "you"?

Reply:
I agree this sounds strange, but I wanted to make it obvious this section was for non-admins until we separate the books.